Source/Microsoft.Teams.Apps.CompanyCommunicator/Models/SentNotificationSummary.cs

```diff
@@ -55,5 +55,34 @@ public class SentNotificationSummary
         /// Gets or sets a value indicating whether the sending process is completed or not.
         /// </summary>
         public bool IsCompleted { get; set; }
+
+        /// <summary>
+        /// Gets or sets the sending started date time.
+        /// </summary>
+        public DateTime SendingStartedDateTime { get; set; }
+
+        /// <summary>
+        /// Gets the value of sending duration in string format.
+        /// </summary>
+        public string SendingDuration
```

Comment:
Let's remove this and have the front end do this calculation.

Reply:
Addressed.